types/relay-runtime/lib/store/RelayModernEnvironment.d.ts

```diff
@@ -41,6 +41,7 @@ export interface EnvironmentConfig {
 export default class RelayModernEnvironment implements Environment {
     options: unknown;
     configName: string | null | undefined;
+    requiredFieldLogger: RequiredFieldLogger;
```

Comment:
Can you clarify Flow source code for this line?  I checked:
https://github.com/facebook/relay/blob/ba9a3f031f8adc0f60bef603a76385433518cd90/packages/relay-runtime/store/RelayModernEnvironment.js

And couldn't find it

Reply:
Without it I get these errors:

```
Property 'requiredFieldLogger' is missing in type 'RelayModernEnvironment' but required in type 'Environment'.
    Type 'RelayModernEnvironment' is not assignable to type 'Environment'.

  node_modules/@types/relay-runtime/lib/store/RelayStoreTypes.d.ts:660:3
    660   requiredFieldLogger: RequiredFieldLogger;
          ~~~~~~~~~~~~~~~~~~~
    'requiredFieldLogger' is declared here.
  node_modules/@types/react-relay/index.d.ts:107:5
    107     environment: IEnvironment;
            ~~~~~~~~~~~
    The expected type comes from property 'environment' which is declared here on type 'IntrinsicAttributes & IntrinsicClassAttributes<ReactRelayQueryRenderer<AnchorListTableQuery>> & Readonly<...> & Readonly<...>'
  node_modules/@types/react-relay/index.d.ts:107:5
    107     environment: IEnvironment;
            ~~~~~~~~~~~
    The expected type comes from property 'environment' which is declared here on type 'IntrinsicAttributes & IntrinsicClassAttributes<ReactRelayQueryRenderer<AnchorListTableQuery>> & Readonly<...> & Readonly<...>'

```

Reply:
Thanks. Found it.  I was looking in the wrong git hash in the Flow types.

Found here:
https://github.com/facebook/relay/blob/v10.1.2/packages/relay-runtime/store/RelayModernEnvironment.js